Acornfile

```diff
@@ -0,0 +1,51 @@
+name:        "Rust Sample Acorn"
+description: "Acorn running a sample Rust app"
+readme:      "./README.md"
+
+
+args: {
+	// Name of the database to create. Defaults to "instance"
+	dbName: "rust-sqlx"
```

Comment:
@aashimodi14 We can also provide custom username & passwords when creating the db service right? Any reason we keep skipping those args?

Reply:
added support for args